packages/database/lib/schema.ts

```diff
@@ -364,6 +364,19 @@ export const daoDetails = pgTable('dao_details', {
   xAccount: varchar('x_account').unique(),
   gitHub: varchar('github').unique(),
   description: text('description'),
+  // Added this in anticipation for web3 auth.
+  creator_acct: pubkey('creator_acct'),
+  // A way for other people to have permissions to make changes.
+  admin_accts: jsonb('admin_accts'),
+  // By initalizing these in the dao details we can reference them on ANY token
+  // which is created through proposals and cascading down through. So the data
+  // will be duplicated, but referencing through this will reduce the burden
+  // of rpc and afford us flexibility with unkown tokens in the UI.
+  // This is a happy medium before we get to onchain data.
+  token_image_url: varchar('token_image_url'),
+  pass_token_image_url: varchar('pass_token_image_url'),
+  fail_token_image_url: varchar('fail_token_image_url'),
+  lp_token_image_url: varchar('lp_token_image_url'),
```

Comment:
That's a fair assumption ummm either way this could be ignored when generating the token (using indexing of on-chain data) so I don't think this should exist as an override unless we can't fetch the data (eg it doesn't exist).

Reply:
And then we should circle back on us loading the image and storing in cloudflare or some other solution vs fetching from request via rpc or even fetching some external place (arweave or whatever) so we are in control of delivery (and thus user XP).